mindsdb/integrations/handlers/duckduckgozeroclick_handler/duckduckgozeroclick_handler.py

```diff
@@ -0,0 +1,98 @@
+import requests
+from typing import Dict, Any
+from collections import OrderedDict
+import pandas as pd
+from mindsdb.integrations.libs.api_handler import APIHandler
+from mindsdb.integrations.libs.response import (
+    HandlerStatusResponse as StatusResponse,
+    HandlerResponse as Response,
+    RESPONSE_TYPE
+)
+from mindsdb.integrations.libs.response import RESPONSE_TYPE
+from mindsdb.integrations.libs.const import HANDLER_CONNECTION_ARG_TYPE as ARG_TYPE
+
+class DuckDuckGoHandler(APIHandler):
+    def __init__(self, name: str = None, **kwargs):
+        super().__init__(name)
+        self.api_key = None
+        self.is_connected = False
+
+        args = kwargs.get('connection_data', {})
+        if 'api_key' in args:
+            self.api_key = args['api_key']
+
+    def connect(self) -> bool:

```

Comment:
How this works? You will need the actually connection here

Reply:
i actually implemented it by looking into binace handler code